model.py

```diff
@@ -8707,7 +8707,7 @@ class Library(Base):
 
     # A short name of this library, to use when identifying it in
     # scripts. e.g. "NYPL" for NYPL.
-    short_name = Column(Unicode, unique=True)
+    short_name = Column(Unicode, unique=True, nullable=False)
```

Comment:
Since this will be used in URLs, should we impose a length limit or a restriction on which characters are valid?